stable/emote/v1/types.proto

```diff
@@ -14,8 +14,7 @@ message EmotePack {
 
 // Data for a single emote.
 message Emote {
-  // The image ID of the emote. This is the ID of the image in the image store
-  // (same place attachments are stored).
+  // The HMC of the emote. Must point to an image.
```

Comment:
does this apply on emote addition / creation as well?

Reply:
It doesn't, file IDs should be accepted on requests, though that needs to be documented